### PR TITLE
Added logs

### PR DESCRIPTION
**Pull request by SmartManoj**
Helps https://github.com/OpenDevin/OpenDevin/issues/2150#issuecomment-2141140899, https://github.com/OpenDevin/OpenDevin/issues/2150#issuecomment-2141164177

### CODE REVIEW

File: opendevin/core/config.py [hunk 1]
```diff
@@ -331,8 +331,8 @@ def load_from_toml(config: AppConfig, toml_file: str = 'config.toml'):
     try:
         with open(toml_file, 'r', encoding='utf-8') as toml_contents:
             toml_config = toml.load(toml_contents)
-    except FileNotFoundError:
-        # the file is optional, we don't need to do anything
+    except FileNotFoundError as e:
+        logger.error(f'Config file not found: {e}')
```
Comment (isavita) [inline opendevin/core/config.py:335]:
```suggestion
        logger.warning(f'Config file not found: {e}')
```

**Comment by SmartManoj** [comment]
Here @isavita?
https://github.com/OpenDevin/OpenDevin/blob/f188abd7a3ddf0b77751184e943078123524dbd4/opendevin/core/config.py#L446-L447

File: opendevin/core/config.py [hunk 2]
```diff
@@ -331,8 +331,8 @@ def load_from_toml(config: AppConfig, toml_file: str = 'config.toml'):
     try:
         with open(toml_file, 'r', encoding='utf-8') as toml_contents:
             toml_config = toml.load(toml_contents)
-    except FileNotFoundError:
-        # the file is optional, we don't need to do anything
+    except FileNotFoundError as e:
+        logger.warning(f'Config file not found: {e}')
```
Comment (li-boxuan) [inline opendevin/core/config.py:335]: No I think this should be a `info`, not a warning. Warning means the user has done something wrong. By design, this config file is optional.

**Comment by SmartManoj** [comment]
> Here @isavita?
> 
> https://github.com/OpenDevin/OpenDevin/blob/f188abd7a3ddf0b77751184e943078123524dbd4/opendevin/core/config.py#L446-L447

cc: @li-boxuan

File: opendevin/core/config.py (hunk 2 above)
Comment (isavita) [inline opendevin/core/config.py:335]:
```suggestion
        logger.info(f'Config file not found: {e}')
```

**Comment by isavita** [comment]
Info sounds better, the warning under was when there is a parsing error.

**Review by neubig — APPROVED** [review]
LGTM, thanks!

**Pull request merged**